arbitrum-docs/node-running/how-tos/data-availability-committee/introduction.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'How to configure a Data Availability Committee: introduction'
+description: Learn what's needed to configure a Data Availability Committee for your chain
+author: jose-franco
+sidebar_label: 'Introduction'
+sidebar_position: 1
+content-type: overview
+---
+
+import PublicPreviewBannerPartial from '../../../partials/_public-preview-banner-partial.md';
+
+<PublicPreviewBannerPartial />
+
+<p>
+  <a data-quicklook-from="arbitrum-anytrust-protocol">AnyTrust</a> chains rely on an external Data
+  Availability Committee (DAC) to store data and provide it on demand instead of using the{' '}
+  <a data-quicklook-from="parent-chain">parent chain</a> as the Data Availability (DA) layer. The
+  members of the DAC run a Data Availability Server (DAS) to handle these operations.
+</p>
+
+This section offers information and a series of how-to guides to help you along the process of setting up a Data Availability Committee. These guides target two audiences: Committee members who wish to deploy a Data Availability Server, and chain owners who wish to configure their chain with the information of the Committee.
+
+Before following the guides in this section, you should be familiarized with how the AnyTrust protocol works and what's the role of the DAC in the protocol. You can find more information about the AnyTrust protocol in [Inside AnyTrust](/inside-anytrust.mdx).
+
+## If you are a DAC member
+
+Committee members will need to run a DAS. To do that, they will first need to generate a pair of keys and deploy a DAS. They may also choose to deploy an additional mirror DAS. Find more information in [How to deploy a DAS](./deploy-a-das.mdx) and [How to deploy a mirror DAS](./deploy-a-mirror-das.mdx).
+
+Here's a basic checklist of actions to complete for DAC members:
+
+- [Deploy a DAS](./deploy-a-das.mdx). Send the following information to the chain owner:
+  - Public BLS key
+  - The https URL for the RPC endpoint which includes some random string (e.g. das.your-chain.io/rpc/randomstring123), communicated through a secure channel
+  - The https URL for the REST endpoint (e.g. das.your-chain.io/rest)
+- [Deploy a mirror DAS](./deploy-a-mirror-das.mdx) if your main DAS doesn't have the REST interface enabled. Send the following information to the chain owner:
```

Comment:
The reason for using a mirror DAS is more along the lines of if you don't want to expose the REST interface of your main DAS to the public. The mirror DAS still needs the REST interface on the DAS to sync from it. The reason deployers might want to do this would be to separate out the server that is handling the Sequencer API from the public REST interface for resiliency, eg in case of a DDoS attack on the REST interface.

Reply:
Alright, I've modified the language to make it clearer than the main use case for running a mirror DAS is to complement your main DAS.